### ISSUE TITLE
Option to allow tabbed pane separator to take full height

### ISSUE THREAD

**Issue by tanmatra** [JFormDesigner/FlatLaf#59]
It may look more "solid", and the latest IDEA is using this style for editor tabs.

![image](https://user-images.githubusercontent.com/708023/74103289-04556080-4b5c-11ea-95db-ed36a7cca777.png)

It can be done simply enough, just by changing `float offset` in line 310 of `FlatTabbedPaneUI` to `0`.

I can create pull request for it.

**Comment by smileatom** [comment]
PR submitted https://github.com/JFormDesigner/FlatLaf/pull/62

**Comment by tanmatra** [comment]
There is a typo:

FlatClientProperties.TABBED_PANE_TAB_SEPARATORS_FULL_HEIGHT = "**JTabbedPane**.tabSeparatorsFullHeight"

but in FlatTabbedPaneUI:

tabSeparatorsFullHeight = UIManager.getBoolean( "**TabbedPane**.tabSeparatorsFullHeight" );

**Comment by DevCharly** [comment]
@tanmatra thanks for the idea. smileatom's PR is now merged into master branch.

To try it out add the latest snapshot (https://github.com/JFormDesigner/FlatLaf#snapshots) to your dependencies and add following to your code (after installing FlatLaf):

~~~java
UIManager.put( "TabbedPane.tabSeparatorsFullHeight", true );
~~~

A new release will be available in some days.

**Comment by DevCharly** [comment]
released in 0.27
